compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/useFormState-dispatch-considered-as-non-reactive.expect.md

```diff
@@ -0,0 +1,57 @@
+
+## Input
+
+```javascript
+import { useFormStatus } from "react-dom";
+
+function Component() {
+  const [formState, dispatchForm] = useFormStatus();
+  const onSubmitForm = () => {
+    dispatchForm();
+  };
+  return <Foo onSubmitForm={onSubmitForm} />;
+}
+
+function Foo() {}
+
+export const FIXTURE_ENTRYPOINT = {
+  fn: Component,
+  params: [],
+};
+
+```
+
+## Code
+
+```javascript
+import { c as _c } from "react/compiler-runtime";
+import { useFormStatus } from "react-dom";
+
+function Component() {
+  const $ = _c(1);
+  const [formState, dispatchForm] = useFormStatus();
+  let t0;
+  if ($[0] === Symbol.for("react.memo_cache_sentinel")) {
+    const onSubmitForm = () => {
+      dispatchForm();
+    };
+
+    t0 = <Foo onSubmitForm={onSubmitForm} />;
+    $[0] = t0;
+  } else {
+    t0 = $[0];
+  }
+  return t0;
+}
+
+function Foo() {}
+
+export const FIXTURE_ENTRYPOINT = {
+  fn: Component,
+  params: [],
+};
+
+```
+      
+### Eval output
+(kind: exception) object is not iterable (cannot read property Symbol(Symbol.iterator))
```

Comment:
hmmm

Reply:
`useFormState` was indeed the correct hook name.